hydra-node/test/Hydra/Chain/Direct/Contract/CollectCom.hs

```diff
@@ -93,7 +93,7 @@ healthyCommits =
  where
   committedUTxO =
     generateWith
-      (replicateM (length healthyParties) genCommittableTxOut)
```

Comment:
should `genCommittableTxOut` not produce only ada utxo?

Reply:
Yeah that is something we could incorporate in this test suite as well. I was just porting what was there before.